packages/components/src/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export type TextAreaProps = {
+  className?: string;
+  placeholder?: string;
+  rows?: number;
+};
+
+const Wrapper = styled.textarea`
+  box-sizing: border-box;
+  width: 100%;
+  min-width: 0;
+  max-width: 100%;
+  height: auto;
+  min-height: 32px;
+  padding: 4px 11px;
+  overflow: auto;
+  font-size: 14px;
+  line-height: 1.5715;
+  color: rgba(0, 0, 0, 0.65);
+  vertical-align: bottom;
+  resize: vertical;
+  border: 1px solid #d9d9d9;
+  border-radius: 2px;
+  font-feature-settings: 'tnum';
+
+  &::placeholder {
+    color: #bfbfbf;
+  }
+
+  &:hover {
+    border-color: #40a9ff;
+    border-right-width: 1px !important;
```

Comment:
Do we really need to use `!important`?

Reply:
I'm not sure, but it was in ant-d sources: https://github.com/ant-design/ant-design/blob/d857800adccdee879d6fbc3fa61fc29e1941966b/components/input/style/mixin.less#L25

Looks like it works OK without it, so removed it.